WarOfEmpires.CommandHandlers/Empires/HealTroopsCommandHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using WarOfEmpires.CommandHandlers.Decorators;
+using WarOfEmpires.Commands.Empires;
+using WarOfEmpires.Domain.Players;
+using WarOfEmpires.Repositories.Players;
+using WarOfEmpires.Utilities.Container;
+
+namespace WarOfEmpires.CommandHandlers.Empires {
+    [InterfaceInjectable]
+    [Audit]
+    public sealed class HealTroopsCommandHandler : ICommandHandler<HealTroopsCommand> {
+        public PlayerRepository _repository;
+
+        public HealTroopsCommandHandler(PlayerRepository repository) {
+            _repository = repository;
+        }
+
+        public CommandResult<HealTroopsCommand> Execute(HealTroopsCommand command) {
+            var result = new CommandResult<HealTroopsCommand>();
+            var player = _repository.Get(command.Email);
+            int staminaToHeal = 0;
+
+            if (!string.IsNullOrEmpty(command.StaminaToHeal) && !int.TryParse(command.StaminaToHeal, out staminaToHeal) || staminaToHeal < 0) {
```

Comment:
The check for empty, and the test for healing with empty stamina to heal, don't make sense in this context to me. It's for troops and workers because you can do one of many, but here there's always just the one so if we make it required the user will see that something's wrong if they empty it.

Reply:
Check for null and empty has been removed after updating the regular expression